server/index.js

```diff
@@ -2,13 +2,13 @@ const express = require('express');
 const path = require('path');
 const router = require('./router.js');
 const port = 3000;
-let app = express();
+
+const app = express();
 
 app.use(express.json());
 app.use(express.static(path.join(__dirname, '../client/dist')))
-// app.get('/home')
-app.use('/api', router)
 
+app.use('/', router)
```

Comment:
We dont need to server any static files right? So we can leave it with just the '/' ? @erikeh